File: app/src/main/java/com/bitquartet/tgscheduler/app/NSApplication.java
package com.bitquartet.tgscheduler.app;

import android.app.AlarmManager;
import android.app.Application;
import android.content.Context;

import com.bitquartet.tgscheduler.utils.ConnectionManager;

public class NSApplication extends Application {

  public Context getContext() {
    return getApplicationContext();
  }

  public AlarmManager getAlarmManager() {
    return (AlarmManager) getApplicationContext().getSystemService(ALARM_SERVICE);
  }
}

File: app/src/main/java/com/bitquartet/tgscheduler/app/NSApplication.kt
package com.bitquartet.tgscheduler.app

import android.app.AlarmManager
import android.app.Application
import android.content.Context

open class NSApplication : Application() {

    open val context: Context
        get() = applicationContext

    open val alarmManager: AlarmManager
        get() = applicationContext.getSystemService(Context.ALARM_SERVICE) as AlarmManager
}
